api/controller/psmdb_cluster.proto

```diff
@@ -60,4 +60,6 @@ message PSMDBClusterParams {
       msg_exists: true
     }
   ];
+  // Paused is true when cluster is suspended.
+  bool paused = 3;
```

Comment:
Do we convert statuses on pmm-managed side?

Reply:
Yes, we do.

Reply:
@AlekSi what do you think? Should we return separate paused field in dbaas-controller API or just new state `PAUSED`?

Reply:
I think we don't need a separate field _or_ a new state: pausing a cluster is _just_ a scaling to zero instances, no? Then I'm not sure why we use a separate CR field, etc